### DISCUSSION TITLE
Reconciliation

### DISCUSSION THREAD

**Discussion by ColGra** [actualbudget/releases#231] [Q&A]
Is there a way to go back to a previous reconciliation point? I have an issue that cannot be resolve unless I step back in time reconciliation-wise.

Colin

**Comment by bpaulien** [comment]
All you have to do is click on the cleared checkmarks down to that date, (removing the checkmarks) and reconcile again with your statement.  Everything that matches your statement should be checked up to that date, and everything beyond that date should not be checked.  

There isn't REALLY a reconciliation, as in there being a status other than clear or unclear, so it's just a matter of marking the transactions that you don't want to show as "reconciled" as being uncleared.

hope that makes sense.

**Reply by MiriamNZ** [reply]
I have a 'reconcile' payee. I add a dummy transaction when i reconcile, and put the $ balance in the notes (in case something earlier has upset things. ).
